model/distro/distro.go

```diff
@@ -574,7 +565,7 @@ func ValidateArch(arch string) error {
 // ValidateCloneMethod checks that the clone mechanism is one of the supported
 // methods.
 func ValidateCloneMethod(method string) error {
```

Comment:
Can this also be moved into the top-level package with the clone methods?

Reply:
Sure, done